apps/discovery_streams/lib/discovery-streams/topic_helper.ex

```diff
@@ -10,7 +10,19 @@ defmodule DiscoveryStreams.TopicHelper do
     |> List.last()
   end
 
+  def get_endpoints() do
+    Application.get_env(:kaffe, :consumer)[:endpoints]
+    |> Enum.map(fn {host, port} -> {to_charlist(host), port} end)
+  end
+
+  def delete_topic(dataset_id) do
+    output_topic = output_topic(dataset_id)
+    Elsa.delete_topic(get_endpoints(), output_topic)
+  end
```

Comment:
This function should probably be called `delete_output_topic` since it is using the output_topic prefix.